java/src/main/java/ai/rapids/cudf/DType.java

```diff
@@ -182,7 +182,9 @@ private DType(DTypeEnum id, int decimalScale) {
 
   /**
    * Returns scale for Decimal Type
-   * @return scale
+   * @return scale If negative, the scale is the number of digits to the right of
```

Comment:
This comment implies that there's some special behavior if the scale is negative vs. >= 0 which is not the case.  It's always the base-10 exponent to multiply the unscaled value to produce the decimal value.  I also think the examples are great and should appear in all the methods that document what the scale parameter does.